### PR TITLE
[stable/grafana] Add option testFramework.imagePullPolicy

### LINKED ISSUE #21608: [stable/grafana] Add option testFramework.imagePullPolicy
**Is your feature request related to a problem? Please describe.**
Yes. I can't change imagePullPolicy of the testFramework pod container.

**Describe the solution you'd like**
Add option testFramework.imagePullPolicy

**Describe alternatives you've considered**
none

**Additional context**
The default `imagePullPolicy: IfNotPresent` might not be a security best practice.

### PR DESCRIPTION
**Pull request by pupimvictor**
Signed-off-by: Victor Pupim <pupimvictor@gmail.com>

#### Is this a new chart
no

#### What this PR does / why we need it:
Add option `testFramework.imagePullPolicy`. The default k8s `imagePullPolicy: IfNotPresent` might not be a security best practice.

#### Which issue this PR fixes
  - fixes #21608

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.0.8
+version: 5.0.9
```
Comment (zanhsieh) [inline stable/grafana/Chart.yaml]: Please bump to the latest version. Maybe 5.0.10 for now.

**Comment by zanhsieh** [comment]
@pupimvictor 
Could you bump the chart again to latest please? Thank you.

**Comment by pupimvictor** [comment]
sure. 5.0.10 or 5.0.11?

**Review by zanhsieh — APPROVED** [review]

**Review by cpanato — APPROVED** [review]
/approve
/lgtm

**Pull request merged**